src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -221,7 +221,7 @@ class TransposeValueTensors_llama3 : public TransposeValueTensors {
 class ScaledDotProductAttentionDecomposition : public ov::pass::MatcherPass {
 public:
     OPENVINO_MATCHER_PASS_RTTI("npuw::LLMCompiledModel::ScaledDotProductAttentionDecomposition");
-    ScaledDotProductAttentionDecomposition() {
+    ScaledDotProductAttentionDecomposition(bool use_high_precision_on_add) {
```

Comment:
not sure if this parameter is required here but if it is, the constructor should be made `explicit`

Reply:
agree while it is not rule one is followed across openvino code base unfortunately